app/aggregates/reviewing/review.rb

```diff
@@ -70,7 +70,7 @@ def mark_as_ready(user_id:)
     on ApplicationReceived do |event|
       @state = :open
       @received_at = event.timestamp
-      @submitted_at = event.data[:submitted_at]
+      @submitted_at = event.data[:submitted_at] || @received_at
```

Comment:
Is this because of returns? Could we make the distinction explicit - use a local variable or a comment perhaps